### DISCUSSION TITLE
is it possible to add a hyperlink to the IP address automatically ?

### DISCUSSION THREAD

**Discussion by Antoine6908** [netbox-community/netbox#11752] [Q&A] [answered]
Hi! 
We use Netbox with many firewalls and it would be great if you could click on the IP address and go to its web interface directly .
thank you in advance for your answer :)

**Comment by jsenecal** [comment]
You can add [markdown ](https://daringfireball.net/projects/markdown/syntax)in the Comments field of an IP address or even use [Custom links](https://blog.sdn.clinic/2021/09/custom-links-in-netbox-shortcut-to-device-webuiipmi/)

Let me know,

**Comment by candlerb** [comment, marked as answer]
Custom Links are probably what you want.

Here is a config I use to display an "Open IPMI" button on any device which has exactly one interface marked "Management Only" with exactly one IP address on that interface.

```
### Link text ###
{% if object.interfaces.filter(mgmt_only=True).count() == 1 and object.interfaces.get(mgmt_only=True).ip_addresses.count() == 1 %}Open IPMI{% endif %}

### Link URL ###
https://{{ object.interfaces.get(mgmt_only=True).ip_addresses.first().address.ip }}/
```
